src/main/webapp/script.js

```diff
@@ -13,145 +13,132 @@
+        const schoolDesc = document.getElementById('school-desc');
+        schoolDesc.innerHTML = '';
+        schoolDesc.append(`${name} is a ${ownership} University 
+            in ${city}, ${state}`);
+
+        // Cost Section.
+        const costDiv = document.getElementById('cost');
+        costDiv.innerHTML = '';
+        costDiv.append(`In-State Tuition: $${inStateTuition}`);
+        costDiv.append(`Out-of-State Tuition: $${outOfStateTuition}`);
+        
+        // Admissions Section.
+        const admissionsDiv = document.getElementById('admissions');
+        admissionsDiv.innerHTML = '';
+        admissionsDiv.append(`Acceptance Rate: ${acceptanceRate}%`);
+        admissionsDiv.append(`Average SAT Score: ${avgSAT}`);
+        admissionsDiv.append(`Average ACT Score: ${avgACT}`);
+
+        // Students Section.
+        const studentsDiv = document.getElementById("students");
+        studentsDiv.innerHTML = '';
+        studentsDiv.append(`Population: ${numStudents} Students`);
+        studentsDiv.append(`4 Year Graduation Rate: ${graduationRate4yr}%`);
+
+        // Draw charts.
+        drawRaceChart(numWhiteStudents, numAsianStudents, numBlackStudents,
+            numHispanicStudents, numIndigenousStudents, numMultiracialStudents, 
+            numUnreportedRaceStudnets);
+        drawGenderChart(numMen, numWomen);
+  });
+}
+
+/**
+ * Inserts the school name into the college scoreboard API link and 
+ * returns the complete link.
+ * @param {string} schoolName
+ */
+function getLink(schoolName) {
+  return ('https://api.data.gov/ed/collegescorecard/v1/schools.json?' +
+      'api_key=C8Uyh2jQCmfjfKN3qwqwcJOi77c5k3V6zM7cRFgJ&school.name=' +
+      `${schoolName}&fields=id,school.city,school.name,school.state,school` +
+      '.school_url,school.ownership,school.minority_serving.' +
+      'historically_black,latest.admissions.admission_rate.overall,' +
+      'latest.admissions.sat_scores.average.overall,latest.admissions' +
+      '.act_scores.midpoint.cumulative,latest.cost.tuition,latest.cost' +
+      '.avg_net_price,latest.student.size,latest.student.demographics' +
+      '.race_ethnicity.white,latest.student.demographics.race_ethnicity' +
+      '.black,latest.student.demographics.race_ethnicity.hispanic,' +
+      'latest.student.demographics.race_ethnicity.asian,latest' +
+      '.student.demographics.race_ethnicity.aian,latest.student.demographics' +
+      '.race_ethnicity.nhpi,latest.student.demographics.race_ethnicity' +
+      '.two_or_more,latest.student.demographics.race_ethnicity' +
+      '.non_resident_alien,latest.student.demographics.race_ethnicity' +
+      '.unknown,latest.student.demographics.men,latest.student' +
+      '.demographics.women,latest.completion.completion_rate_4yr_100nt,' +
+      'school.main_campus,school.institutional_characteristics.level');
+}
+
+// Load the Visualization API and the corechart package.
+google.charts.load('current', {'packages':['corechart']});
```

Comment:
These calls should probably happen as part of the `loadSchoolInfo`  function instead of here, to run when the file is evaluated.

Reply:
I tried that however it only works when they are left out of the functions.

Reply:
Ah okay, no worries.